icons/cyberdefenders.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>CyberDefenders</title><path d="M18.954 17.477c-.13 2.73-2.392 4.69-5.404 4.69H7.298v-6.103l-2.192-.96V24h8.44c4.363 0 7.589-2.964 7.589-6.875 0-.056-.008-.107-.01-.162a9.705 9.642 0 0 1-2.17.514zm1.368-9.489C18.648 4.32 17.764.996 15.45.213c-1.345-.454-2.008-.086-2.657.219a.3.297 0 0 0-.07.497c.368.307.813.696 1.349 1.18-1.96-1.133-7.375-2.83-7.927-.773-.499 1.857-1.33 4.6-1.33 4.6S.535 5.495.042 7.302c-.492 1.81 3.346 5.282 9.697 7.141 6.35 1.86 11.271 1.37 13.388-.929 2.118-2.3.07-5.472-2.806-5.526zm-1.26.03c-.487.475-2.084 1.68-5.59 1.43-3.752-.27-6.858-2.48-7.642-3.087a.373.37 0 0 1-.136-.39c.047-.185.128-.473.247-.842a.385.383 0 0 1 .552-.22c1.588.902 4.125 2.227 6.827 2.404 2.416.16 3.867-.54 4.81-1.06a.377.375 0 0 1 .537.19l.477 1.16a.386.384 0 0 1-.082.413z"/></svg>
```

Comment:
Still seeing [slight differences](https://github.com/simple-icons/simple-icons/assets/1830867/381bb00e-83bb-4a8a-aee2-f672f5a280cf) (see the edges) between your PR and the source icon. Below icon seems to be matching at 3 decimals of precision, you're welcome to use it - but we'll need another reviewer to sign off on the PR.

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>CyberDefenders</title><path d="M18.918 17.48c-.126 2.727-2.384 4.696-5.364 4.696H7.34v-6.123l-2.185-.957V24h8.381c4.334 0 7.549-2.962 7.549-6.881v-.163c-.65.235-1.372.415-2.167.524Zm1.355-9.501C18.611 4.313 17.726.989 15.432.213c-1.336-.452-2.005-.091-2.637.217-.199.09-.235.361-.072.505.361.307.813.687 1.336 1.174-1.95-1.138-7.333-2.835-7.874-.776-.488 1.86-1.319 4.587-1.319 4.587S.603 5.487.116 7.293c-.488 1.806 3.323 5.274 9.627 7.134 6.303 1.861 11.198 1.373 13.311-.921 2.113-2.294.072-5.473-2.781-5.527Zm-1.247.036c-.487.47-2.077 1.68-5.563 1.427-3.738-.271-6.809-2.474-7.604-3.088-.126-.091-.18-.235-.126-.398.054-.18.126-.469.253-.849.072-.234.343-.343.542-.216 1.571.903 4.1 2.221 6.791 2.402 2.402.163 3.847-.542 4.786-1.066.199-.108.452-.018.542.199l.47 1.156c.036.162.018.325-.091.433Z"/></svg>
```